include/inviwo/core/datastructures/image/image.h

```diff
@@ -46,6 +47,11 @@ namespace inviwo {
  */
 class IVW_CORE_API Image : public DataGroup<Image, ImageRepresentation>, public MetaDataOwner {
 public:
+    using ColorLayerIterator =
+        util::IndirectIterator<std::vector<std::shared_ptr<Layer>>::iterator>;
+    using const_ColorLayerIterator =
```

Comment:
I would stick to camel case
```suggestion
    using ConstColorLayerIterator =
```